src/MSBuild.Bootstrap/MSBuild.Bootstrap.csproj

```diff
@@ -45,10 +45,10 @@
   </ItemGroup>
 
   <!-- Use deps file from this project with additional dependencies listed instead of the one generated in the MSBuild project -->
-  <Target Name="UpdateMSBuildDepsFile" AfterTargets="Build" Condition="'$(TargetFramework)' == 'netcoreapp2.1' or '$(TargetFramework)' == 'netstandard2.0'">
+  <Target Name="UpdateMSBuildDepsFile" AfterTargets="Build" Condition="'$(TargetFramework)' == 'net5.0' or '$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
Compat property functions (and next target condition)

Reply:
I tried using `"$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net5.0')) or $([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'netstandard2.0'))` and hit issues with net472:
```
C:\src\git\msbuild\src\MSBuild.Bootstrap\MSBuild.Bootstrap.csproj(49,5): error MSB3030: Could not copy the file "C:\src\g
it\msbuild\artifacts\bin\MSBuild.Bootstrap\Debug\net472\MSBuild.Bootstrap.deps.json" because it was not found.
```

Should I be using a different intrinsic function here?

Reply:
Ah, unfortunate. That's right but `net472` _is_ compatible with `netstandard2.0` so it's a bit more complicated. Can it be just compatible with `netcoreapp2.1` to catch our core flavors? We're not building netstandard anything anymore right/

Reply:
As I just figured out in the other thread in this PR, yeah there are still projects building netstandard2.0.  Looks like it's just Microsoft.Build.Tasks.csproj:

![image](https://user-images.githubusercontent.com/4691428/104521091-1f70f180-55b1-11eb-9b46-f212e3dbb896.png)

Reply:
I don't think there's a good reason to not retarget that. But it doesn't have to be in this PR. Reasons to do it: use new APIs etc in tasks, which is a good thing.

Reply:
I quickly updated my comment after noticing my query was looking for the one project. There are many that target netstandard2.0. Seems like this should be tracked and figured out in a different issue.

Reply:
Sounds like a plan but let's prioritize that soon after this so we don't forget.

Reply:
Just to clarify, you want to update them to `netstandard2.1`?

Reply:
Tracking here: https://github.com/dotnet/msbuild/issues/6032

The new condition definitely doesn't look any cleaner than before, but it's more future-proof.